app/services/custom_report_service.py

```
@@ -1072,11 +1072,16 @@ def _generate_trend_charts(
                 )
             )
         for lc in trend_charts.lab_test_charts:
+            label = f"{lc.test_name} ({lc.unit})" if lc.unit else lc.test_name
             chart_tasks.append(
                 (
-                    lc.test_name,
+                    label,
                     lambda l=lc: fetcher.fetch_lab_test_trend(
```

Ruff (0.15.10)

[error] Ambiguous variable name: `l`

(E741)